.gitmodules

```diff
@@ -0,0 +1,13 @@
+[submodule "scripts/word_embeddings/CLI11"]
+	path = scripts/word_embeddings/extern/CLI11
+	url = https://github.com/CLIUtils/CLI11.git
+[submodule "scripts/word_embeddings/cnpy"]
+	path = scripts/word_embeddings/extern/cnpy
+	url = https://github.com/leezu/cnpy
```

Comment:
There is a bug in the original project. It does not support Zip64 standard, meaning that if the numpy archive is > 4GB it will write a invalid file.. I just replace the low-level zlip with the high-level libzip in https://github.com/rogersce/cnpy/pull/41

Reply:
Ie. libzip handles all these Zip-standard details for us.